_posts/2022-03-24-opensearch-java-runtime.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Using Different Java Runtimes with OpenSearch"
+authors:
+  - dblock
+  - smortex
+  - reta
+  - ryanbogan
+date: 2022-03-24
+categories:
+  - technical-post
+twittercard:
+  description: "OpenSearch ships with a bundled Java Development Kit (JDK) that has recently been updated to version 11 (LTS). In this blog post we'll explain this change, and describe new features that make swapping the Java runtime easier."
+excerpt: ""
```

Comment:
Needs an excerpt or it doesn't render nicely on the homepage

![Screen Shot 2022-03-23 at 3 27 32 PM](https://user-images.githubusercontent.com/1152927/159798716-cf0a16b1-a28c-45dc-8505-189c9d1f3aef.png)